src/styles/login.css

```diff
@@ -1,55 +1,72 @@
 @import "main.css";
 
-body{
-    margin: 0; /*why?*/
+.wholeLogin {
+  display: flex;
+  align-items: center;
+  min-height: 100vh;
 }
 
-.wholesignup{
-    margin-top: 0;
+.loginBanner {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  flex-basis: 40%;
+  height: 100vh;
+  background: var(--tertiaryGreen);
+  box-sizing: border-box;
 }
 
+.loginBannerImage {
+  position: relative;
+  width: 100%;
+  max-height: 120px;
+  object-fit: cover;
+  object-position: center;
+}
 
-.imgLogo{
-position: relative;
-bottom: 10%;
-margin: auto;
-width: 50%;
+.loginText,
+.signupText {
+  color: var(--mainGreen);
+  font-size: min(3vw, 1.7rem);
+  font-weight: bolder;
```

Comment:
What about 800 for the font weight and we have a global setting for the font family in main.css 
```css
.loginText,
.signupText {
  color: var(--mainGreen);
  font-size: min(3vw, 1.7rem);
  font-weight: 800;
  align-items: center;
  margin-inline: 30px;
  font-family: var(--mainFontFamily);
}

```